src/components/progress-bar/progress-bar.component.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./progress.styles.scss";
+
+const ProgressBar = ({ children, barType, stat }) => {
+    const barStyle = {
+        width: `${stat}%`,
+    };
+    return (
+        <div className="container">
+            <div className="progress">
+                {children}
+                <div className={`bar ${barType}`} style={barStyle}></div>
```

Comment:
<div className={`bar ${barType}`} style={barStyle} /> is enough, there's no content inside